cluster/schema/meta_class.go

```diff
@@ -478,6 +482,25 @@ func (m *metaClass) applyShardProcess(name string, action command.TenantProcessR
 
 		if count == len(processes) {
 			copy.Status = req.Tenant.Status
+			// TODO ask loic for thoughts on how to move this out of meta class? Also does that need to be in the metaclass ?
```

Comment:
@reyreaud-l or @moogacs would love your thoughts here for a follow up PR

Reply:
The one thing I have in mind is that we can move it to schema/manager.go -> in `UpdateTenants` we do 
```
319     return s.apply(
320         applyOp{
321             op: cmd.GetType().String(),
322             // updateSchema func will update the request's tenants and therefore we use it as a filter that is then sent
323             // to the updateStore function. This allows us to effectively use the schema update to narrow down work for
324             // the DB update.
325             updateSchema: func() error { return s.schema.updateTenants(cmd.Class, cmd.Version, req) },
326             updateStore:  func() error { return s.db.UpdateTenants(cmd.Class, req) },
327             schemaOnly:   schemaOnly,
328         },
329     )
```

We can probably add a `notifyFunc` as a parameter to `applyOp` that is used to send the tenants (in a non blocking way) to the querier nodes.

Reply:
curious why not in this `meta_class.go` itself? why have to move?

Reply:
https://github.com/weaviate/weaviate/pull/5853#discussion_r1805393731